Unix/buildtool

```diff
@@ -776,6 +779,8 @@ if [ "$arg1" = "cflags" -o "$arg1" = "cxxflags" ]; then
                 fi
             fi
 
+			test -n "$gcov_opt" && r="$r -ftest-coverage -fprofile-arcs -lgcov"
```

Comment:
Not sure you need the -lgcov here as that us a link option and not compile options.